### ISSUE TITLE
Error on pre-commit hook. 

### ISSUE THREAD

**Issue by osirvent** [blitz-js/blitz#496]
### What is the problem?
Unable to create initial commit on a new blitz app

### Steps to Reproduce: 

1. Create a new blitz app

### Versions:

```
macOS Catalina | darwin-x64 | Node: v12.16.2

blitz: 0.9.1 (global)
blitz: 0.9.1 (local)
```

### Supporting Documentation
<img width="763" alt="imagen" src="https://user-images.githubusercontent.com/5927133/81592457-4c378e00-93be-11ea-9420-35bdd304b639.png">

Git output
```
✖ eslint --fix:
Error: Failed to load parser '@typescript-eslint/parser' declared in '.eslintrc.js » eslint-config-react-app#overrides[0]': Cannot find module 'json-schema'
```

Please include applicable logs and screenshots that show your problem.

**Comment by wKovacs64** [comment]
This is an upstream [issue](https://github.com/typescript-eslint/typescript-eslint/issues/2009) with `@typescript-eslint/eslint-plugin@2.32.0`. They have a fix merged already, but have not released at the time I'm writing this comment.

**Comment by osirvent** [comment]
Yeah! Downgrading those to `2.31.0` solve the commit error

```
"@typescript-eslint/eslint-plugin": "2.31.0",
"@typescript-eslint/parser": "2.31.0",
```

**Comment by osirvent** [comment]
@flybayer The fix has been released to latest - 2.33.0

**Comment by flybayer** [comment]
@osirvent great, thanks!
